sbin/umount/umount.c

```diff
@@ -637,7 +661,47 @@ usage(void)
 {
 
 	(void)fprintf(stderr, "%s\n%s\n",
-	    "usage: umount [-fNnv] special ... | node ... | fsid ...",
-	    "       umount -a | -A [-F fstab] [-fnv] [-h host] [-t type]");
+	    "usage: umount [-dfNnv] special ... | node ... | fsid ...",
+	    "       umount -a | -A [-F fstab] [-dfnv] [-h host] [-t type]");
 	exit(1);
 }
+
+int
+md_detach(const char *device)
+{
+	struct md_ioctl mdio;
+	char *p;
+	int fd;
+
+	bzero(&mdio, sizeof(mdio));
+
+	mdio.md_version = MDIOVERSION;
+	mdio.md_options = fflag ? MD_FORCE : 0;
+
+	if (strncmp(device, DEV_MD, sizeof(DEV_MD) - 1)) {
+		if (!multiple)
```

Comment:
We checked this before entering the function

Reply:
sorry, i meant for multiple

Reply:
> sorry, i meant for multiple

In this case we need the check to avoid a warning when using `-a`.